injected/src/features/autofill-password-import.js

```diff
@@ -157,13 +308,33 @@ export default class AutofillPasswordImport extends ContentFeature {
      * @param {string} path
      */
     async handleElementForPath(path) {
+        this.removeOverlayIfNeeded();
         const supportedPaths = [this.#exportButtonSettings?.path, this.#settingsButtonSettings?.path, this.#signInButtonSettings?.path];
-        if (supportedPaths.indexOf(path)) {
+        if (supportedPaths.includes(path)) {
             try {
-                const { element, style, shouldTap } = (await this.getElementAndStyleFromPath(path)) ?? {};
+                const { element, style, shouldTap, shouldWatchForRemoval } = (await this.getElementAndStyleFromPath(path)) ?? {};
+                this.setElementToCenterOn(element, style);
                 if (element != null) {
-                    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-                    shouldTap ? this.autotapElement(element) : this.animateElement(element, style);
+                    if (shouldTap) {
+                        this.autotapElement(element);
+                    } else {
+                        const domLoaded = new Promise((resolve) => {
+                            if (document.readyState === 'loading') {
+                                document.addEventListener('DOMContentLoaded', resolve);
```

Comment:
Also we could pass `once` here. Not really a problem just might promote it's use for when it's more needed.

Reply:
Sorry not sure I fully understood but I suppose https://github.com/duckduckgo/content-scope-scripts/pull/1184/commits/52362406bd13197f731571649ade03c91f79b5fa